src/table.rs

```diff
@@ -73,6 +100,19 @@ where
         self.len() == 0
     }
 
+    /// Checks if the suffix table is lexicographically sorted. This is always true for valid suffix tables.
+    pub fn is_sorted(&self) -> bool {
+        if self.table.len() > 100_000 {
```

Comment:
At least some of rayon's functions automatically short circuit to a serial algorithm for small inputs, so this may be unnecessary but whatever.

Reply:
This was faster in testing although I can't remember how large the performance gain was

Reply:
Will probably remove this